src/core/compact_object.h

```diff
@@ -445,13 +446,22 @@ class CompactObj {
     };
   } __attribute__((packed));
 
+  struct JsonConsT {
+    JsonType* json_ptr;
+    // Owning and must be explicitly deallocated
+    MiMemoryResource* mr;
```

Comment:
I am not sure why you keep `MiMemoryResource* mr;` inside each CompactObject.
1. JsonType already has allocator in it
2. All json objects will use the same MiMemoryResource instance unless I am missing something

Reply:
I know about this. I keep it because I want to `DCHECK` the hell out of it. It's just breaks with `pmr` in so many levels that even if we rolled in production I would add `LOG(warning)` to warn us that the `pmr` went brrr